SB2018-volunteer/2018-04-13.md

```diff
@@ -0,0 +1,36 @@
+---
+to: All Volunteers
+subject: Volunteer Information - SourceBots 2018
+---
+
+First of all, on behalf of everyone at SourceBots we would like to say a **massive** thank you for volunteering at the competition, even though it will be hard work, we can assure you that it will be a fun weekend filled with robot madness and endless fun.
+
+To that end, we thought now would be a good time to send an email detailing information about the competition!
+
+### Date and location
+
+As a reminder, the competition dates are 21st April - 22nd April and it will be held in [The Cube (SUSU Building)](http://data.southampton.ac.uk/building/42.html) which is on the bottom floor of the Students' Union building.
```

Comment:
I suggest linking to our website instead. There's more information there, and it also links to the cube.

Reply:
The cube is not on the bottom floor!